tests/src/Kernel/Controller/ViewerControllerTest.php

```diff
@@ -76,12 +76,13 @@ public function testEditorCollaboraUnavailable(): void {
       $this->assertBadRequestResponse(
         'The Collabora Online editor/viewer is not available.',
         $request,
-        [
-          'message' => "Collabora Online is not available.<br>\n" . Error::DEFAULT_ERROR_MESSAGE,
-          'level' => RfcLogLevel::WARNING,
-        ],
         $name
       );
+      $this->assertLogMessage(
+        RfcLogLevel::WARNING,
+        "Collabora Online is not available.<br>\n" . Error::DEFAULT_ERROR_MESSAGE,
+        assertion_message: $name
+      );
```

Comment:
We should assert that the original message is part of the log.
```suggestion
      $this->assertLogMessage(
        RfcLogLevel::WARNING,
        "Cannot show the viewer/editor.<br>\n" . Error::DEFAULT_ERROR_MESSAGE,
        [
          '%type' => CollaboraNotAvailableException::class,
          '@message' => 'The Collabora Online connection is not configured.',
        ],
        assertion_message: $name
      );
```
(I think '%type' and '@message' are enough. We don't need %function, %line and %file.)

(also do this for the other case)